source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/SourceCharacteristics.java

```diff
@@ -23,8 +24,11 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
 
+import nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSSourceCharacteristics;
```

Comment:
Not sure if package cycles are a problem, but this import does introduce one.

Reply:
I'll move the class to avoid cycle.